source/sass/mixins/_decorations.scss

```diff
@@ -2,7 +2,7 @@
 @import "../variables/color";
 @import "../variables/grid";
 
-@mixin divider($dimension) {
+@mixin border($dimension) {
```

Comment:
We should extract this change into a separate PR and merge first. (Spec name also needs updating.)